flocker/node/test/test_deploy.py

```diff
@@ -1512,7 +1455,8 @@ def test_open_port_needs_creating(self):
         desired = Deployment(nodes=nodes)
         result = api.calculate_changes(
             desired_configuration=desired,
-            current_cluster_state=DeploymentState(nodes=[node_state]))
+            current_cluster_state=DeploymentState(nodes=[node_state]),
```

Comment:
Here an in later cases where cluster state is not empty, local state being EMPTY_LOCAL_STATE is often incorrect and not how real code would work. Specifically the cases where the node for whom state is known is the on the Deployer is for.